doc/dune-files.rst

```diff
@@ -1988,40 +1964,43 @@ The loading of the plugin is done using the facilities generated by
 generate_sites_module (Since 2.8)
 ---------------------------------
 
-Dune proposes some facilities for dealing with :ref:`sites<sites>` in a program. The
-``generate_sites_module`` stanza will generate code for looking up the correct locations
-of the sites' directories and for loading plugins. It works after installation
-with or without the relocation mode, inside Dune rules, and when using Dune executables.
-For promotion, it works only if the generated modules are solely in the executable (or
-library statically linked) promoted; generated modules in plugins won't work.
+Dune proposes some facilities for dealing with :ref:`sites<sites>` in a program.
+The ``generate_sites_module`` stanza will generate code for looking up the
+correct locations of the sites' directories and for loading plugins. It works
+after installation with or without the relocation mode, inside Dune rules, and
```

Comment:
No commas necessary in this sentence. Just:
…with or without the relocation mode inside Dune rules and when using…

Reply:
I think that is because it is a list so the commas might be needed.